db/migrations/60_merge_users_items_into_groups_attempts.sql

```diff
@@ -31,6 +31,11 @@ FROM `users_items`
 
 ALTER TABLE `groups_attempts` DROP INDEX `group_id_item_id_order`;
 
+DELETE `users_items`
+    FROM `users_items`
+    LEFT JOIN `items` ON `items`.`id` = `users_items`.`item_id`
+    WHERE `items`.`id` IS NULL;
+
```

Comment:
So your "JOIN `items` ON `items`.`id` = `users_items`.`item_id`" in the next update which just there to exclude non existing items (right?) can disappear. 
I think doing the same with `users` may make sense as well (even if I see no rows matching in the current dump)

Reply:
fixed